x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -61,6 +63,38 @@ export const getPackagePoliciesHandler: RequestHandler<
   }
 };
 
+export const bulkGetPackagePoliciesHandler: RequestHandler<
+  undefined,
+  undefined,
+  TypeOf<typeof BulkGetPackagePoliciesRequestSchema.body>
+> = async (context, request, response) => {
+  const soClient = (await context.core).savedObjects.client;
+  const { ids, ignoreMissing } = request.body;
+  try {
+    const items = await packagePolicyService.getByIDs(soClient, ids, {
+      ignoreMissing,
+    });
+
+    if (!items) {
+      throw new Error('Invalid results from packagePolicyService.getByIDs');
```

Comment:
Why do we need to throw an error if there are no `items`?

Reply:
going to change that to return an empty array